native/src/components/CalendarChoiceModal.tsx

```diff
@@ -20,26 +20,13 @@ const StyledList = styled(FlatList as typeof FlatList<Calendar>)`
   flex-grow: 0;
 `
 
-const ButtonTitle = styled.Text`
-  font-family: ${props => props.theme.legacy.fonts.native.decorativeFontBold};
-  color: ${props => props.theme.legacy.colors.textColor};
-`
-
-const ButtonDescription = styled.Text`
-  font-family: ${props => props.theme.legacy.fonts.native.contentFontRegular};
-  color: ${props => props.theme.legacy.colors.textSecondaryColor};
-`
-
 const Divider = styled.View`
   background-color: ${props => props.theme.legacy.colors.textDecorationColor};
   height: 1px;
   margin: 8px 0;
 `
 
-const StyledText = styled.Text`
-  font-family: ${props => props.theme.legacy.fonts.native.decorativeFontRegular};
-  color: ${props => props.theme.legacy.colors.textColor};
-`
+type ExportType = 'only-this-event' | 'all-future-events'
```

Comment:
🙃 It would be possible to just convert a boolean value to a string (`'true'`/`'false'`) as well. Not sure if that would be better though.